src/transformState.js

```diff
@@ -5,7 +5,32 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const changes = Object.values(actions);
+
+  const toRemove = [];
+
+  for (let i = 0; i < changes.length; i++) {
```

Comment:
You can just do like that:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of
```suggestion
  for (const action of actions) {
```